docs/csharp/language-reference/compiler-options/advanced.md

```diff
@@ -273,3 +274,22 @@ Specifies the name of an assembly whose non-public types a *.netmodule* can acce
 - The existing assembly has granted friend assembly access to the assembly into which the .*netmodule* will be built.
 
 For more information on building a .netmodule, see [**TargetType**](output.md#targettype) option of **module**. For more information on friend assemblies, see [Friend Assemblies](../../../standard/assembly/friend.md).
+
+## ReportIVTs
+
+Enable or disable additional diagnostic information about <xref:System.Runtime.CompilerServices.InternalsVisibleToAttribute?displayProperty=nameWithType> attributes found during the compilation:
+
+```xml
+<ReportIVTs>true</ReportIVTs>
```

Comment:
What is the correct hierarchy for the `ReportIVTs` tag? Is it top-level, or nested, and if it's nested, we should really help the reader by showing it with the appropriate nesting.

Reply:
added at the top of the, as it's the same for all elements.